### PR TITLE
Now launch with is only shown if the selected version is valid

### PR DESCRIPTION
**Pull request by agduncan94**
The launch with section is only shown if the version selected is valid.
If it isn't then a message is shown.

Deals with https://github.com/ga4gh/dockstore/issues/1303

### CODE REVIEW

File: src/app/container/launch/launch.component.html [hunk 1]
```diff
@@ -15,45 +15,53 @@
...
-        <select id="launchWithDescriptor" class="form-control input-sm" [(ngModel)]="currentDescriptor" (change)="reactToDescriptor()">
-          <option *ngFor="let descriptor of validDescriptors" [ngValue]="descriptor.value">
-            {{ descriptor.value }}
-          </option>
-        </select>
-      </div>
+  <div *ngIf="!_selectedVersion.valid">
+    <label tooltip="Commands to launch tool from the command line">Launch With</label>
+    <div class="alert alert-warning" role="alert">
+      This section is only available for valid versions.
```
Comment (denis-yuen) [inline src/app/container/launch/launch.component.html]: For this alert, we should probably say what the user can do to fix the situation (i.e. provide valid descriptors).
For workflows at least, still undecided about tools

File: src/app/workflow/launch/launch.component.html [hunk 2]
```diff
@@ -14,37 +14,45 @@
...
     <label tooltip="Commands to launch workflow from the command line">Launch With</label>
-  </div>
-  <div class="well">
-    <div class="well well-sm" tooltip="Commands for creating a runtime JSON template">
-      Make a runtime JSON template and fill in desired inputs, outputs, and other parameters
-      <pre>{{ params }}</pre>
-      or grab one that the workflow author has provided (if applicable)
-      <pre>{{ wgetTestJsonDescription }}</pre>
+    <div class="alert alert-warning" role="alert">
+      This section is only available for valid versions.
```
Comment (denis-yuen) [inline src/app/workflow/launch/launch.component.html]: Same here

File: src/app/container/launch/launch.component.html (hunk 1 above)
Comment (denis-yuen) [reply src/app/container/launch/launch.component.html]: Undecided, maybe we explain and vote in stand-up

Comment (agduncan94) [reply src/app/container/launch/launch.component.html]: Ok so this is fine to merge with just the workflow change, and we can add the tool comment at a later time?

**Pull request merged**